lib/handleRequest.js

```diff
@@ -21,7 +21,7 @@ function handleRequest (err, request, reply) {
 
   var contentType = headers['content-type']
 
-  if (method === 'POST' || method === 'PUT' || method === 'PATCH') {
+  if (method === 'POST' || method === 'PUT' || method === 'PATCH' || method === 'TRACE') {
```

Comment:
CONNECT is omitted here intentionally?

Reply:
Yup. CONNECT requests do not actually have a body ([see MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/CONNECT#:~:text=Request%20has%20body)). They establish a long-lived connection instead which should never arrive in the body parser.